static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -46,8 +46,6 @@ const METRIC_CONDITION_MAP = {
   [MetricValues.USERS]: UNIQUE_USER_FREQUENCY_CONDITION,
 } as const;
 
-const DEFAULT_PLACEHOLDER_VALUE = '10';
```

Comment:
since this is a string and it is being used in 2x different places, shouldn't we keep it in a constant so it is easier to update if we want to change this value?

Reply:
hmm, my thinking was that default value and placeholder are not necessarily the same value + it is not critical that they are the same.
e.g. we may want to have a placeholder text like "Enter a number..." in the future
That's why I hardcoded the values to improve readability. 🤔